test/common/stream_info/test_util.h

```diff
@@ -16,7 +16,11 @@ class TestStreamInfo : public StreamInfo::StreamInfo {
     memset(&fake_time, 0, sizeof(fake_time));
     fake_time.tm_year = 99; // tm < 1901-12-13 20:45:52 is not valid on macOS
     fake_time.tm_mday = 1;
-    start_time_ = std::chrono::system_clock::from_time_t(timegm(&fake_time));
+#if !defined(WIN32)
+    start_time_ = std::chrono::system_clock::from_time_t(::timegm(&fake_time));
```

Comment:
This is a particular problem... There are likely more portable solutions which don't violate the check_validate logic that if failing on the presence of a timegm() call. We simply used the existing logic and would beg another eager volunteer to take on that particular refactoring using stdc++ or absl api, no win32-internals knowledge required.

Reply:
@jmarantz I think we've been discussing use absl::Time and move away from chrono, will that ultimately solve this issue as well?

Reply:
I would be in favor of moving to absl::Time. However it's a ton of work. I count 1716 std::chrono refs in the codebase.

We'd probably have to do it in a few refactors.

Reply:
Can't we `sed` this migration? Maybe with some wrappers?

Reply:
Is this item worth it's own issue, maybe? I'd beg for this bit to be accepted in the short-term.

Reply:
Yes, it's definitely a distinct issue. Can you open a GH issue and tag this as TODO?